pkgs/top-level/all-packages.nix

```
@@ -2566,8 +2566,15 @@ with pkgs;
 
   microcom = callPackage ../applications/terminal-emulators/microcom { };
 
-  mlterm = darwin.apple_sdk_11_0.callPackage ../applications/terminal-emulators/mlterm {
-    inherit (darwin.apple_sdk_11_0.frameworks) Cocoa;
+  mlterm = callPackage ../applications/terminal-emulators/mlterm { };
+  mlterm-wayland = mlterm.override {
+    guiChoices = {
+      wayland = true;
+    };
+    typeEnginesChoices = {
+      cairo = true;
+    };
```

Cairo is on by default. You'd want to disable the legacy ones, right?

```suggestion
    typeEnginesChoices = {
      xcore = false;
      xfg = false;
    };
```

---

Had no idea `xcore` is considered legacy, good to know. Disabled it.